src/main/proto/BasicTypes.proto

```diff
@@ -80,15 +80,43 @@ message AccountAmount {
     sint64 amount = 2; // The amount of tinybars (for Crypto transfers) or in the lowest denomination (for Token transfers) that the account sends(negative) or receives(positive)
 }
 
+message SerialNumbers {
+    repeated int64 serialNumbers = 1; //Applicable to tokens of type NON_FUNGIBLE. The list of serial numbers to be burned/wiped.
+}
+
+message AmountOrSerialNumbers {
+    oneof amountOrSerialNumbers {
+        int64 amount = 1; //Applicable to tokens of type FUNGIBLE. Amount of fungible tokens to burn/wipe
+        SerialNumbers serialNumbers = 2; //Wrapper for serial numbers array
+    }
+}
+
 /* A list of accounts and amounts to transfer out of each account (negative) or into it (positive). */
 message TransferList {
     repeated AccountAmount accountAmounts = 1; // Multiple list of AccountAmount pairs, each of which has an account and an amount to transfer into it (positive) or out of it (negative)
 }
 
+message AccountAmounts {
+    repeated AccountAmount transfers = 1; // Multiple list of AccountAmounts, each of which has an account and amount
+}
+
+message NftTransfer {
+    AccountID senderAccountID = 1;  // Sending account
+    AccountID receiverAccountID = 2;  // Receiving account
+    int64 serialNumber = 3;  // Serial number that is being transferred
+}
+
+message NftTransfers {
+    repeated NftTransfer nftTransfers = 1; // Applicable to tokens of type NON_FUNGIBLE
+}
+
 /* A list of token IDs and amounts representing the transferred out (negative) or into (positive) amounts, represented in the lowest denomination of the token */
 message TokenTransferList {
     TokenID token = 1; // The ID of the token
-    repeated AccountAmount transfers = 2; // Multiple list of AccountAmounts, each of which has an account and amount
```

Comment:
I think this has to be deprecated rather than deleted
@Daniel-K-Ivanov

Reply:
1) Yes, I think that we should deprecate it
2) I think that we should have `repeated AccountAmount transfers =2` compared to `AccountAmounts transfers = 2;`

For reference:
https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-17.md#tokentransferlist

Reply:
`oneof fields cannot have labels.` Therefore I wrapped it in a separate message and placed it where it is now.